test/v2/controllers/V2MovementsControllerSpec.scala

```diff
@@ -3043,25 +3044,88 @@ class V2MovementsControllerSpec
   "POST /movements/:movementId/messages/:messageId" - {
 
     "should return Ok when response from upscan is valid" - {
-      "and uploading to object-store succeeds" in forAll(arbitraryMovementId.arbitrary, arbitraryMessageId.arbitrary, arbitraryObjectSummaryWithMd5.arbitrary) {
-        (movementId, messageId, objectSummary) =>
-          when(
-            mockObjectStoreService.addMessage(any[String].asInstanceOf[DownloadUrl], any[String].asInstanceOf[MovementId], any[String].asInstanceOf[MessageId])(
-              any(),
-              any()
+
+      "and uploading to object-store succeeds" - {
```

Comment:
Please combine these 2 lines with  ``` should return Ok when response from upscan is valid ``` and all should come together

Reply:
No, this separation makes sense -- we will always return Ok to upscan if the response from upscan is schema valid, even if we then can't upload to object store. Returning Ok to upscan doesn't mean that everything succeeded, just that the upscan response was one we could read.